wled00/data/settings_wifi.htm

```diff
@@ -136,12 +136,75 @@
 			getLoc();
 			loadJS(getURL('/settings/s.js?p=1'), false);	// If we set async false, file is loaded and executed, then next statement is processed
 			if (loc) d.Sf.action = getURL('/settings/wifi');
+			setTimeout(tE, 500); // wait for DOM to load before calling tE()
+		}
+
+		var rC = 0; // remote count
+		// toggle visibility of ESP-NOW remote list based on checkbox state
+		function tE() {
+			const e = d.querySelector('input[name="RE"]').checked;
+			const l = gId('rlc');  // remote list container
+			// keep the hidden input with MAC addresses, only toggle visibility of the list UI
+			if (l) l.style.display = e ? 'block' : 'none';
+		}
+		// reset remotes: initialize empty list (called from xml.cpp)
+		function rstR() {
+			gId('rml').innerHTML = ''; // clear remote list
+			rC = 0;
+			uR(); // update remote list
+		}
+		// add remote MAC to the list
+		function aR(id, mac) {
+			if (!/^[0-9A-F]{12}$/i.test(mac)) return; // check for valid hex string
+			if (rC >= 10) return;
+			let inputs = gId('rml').getElementsByTagName('input');
+			for (let i = 0; i < inputs.length; i++) {
+				if (inputs[i].value === mac) return;
+			}
+			let list = gId('rml');
```

Comment:
Reorder and reuse `list` object. With shorter name even more bytes could be saved.